static/js/components/tiles/map_tile.tsx

```diff
@@ -449,7 +526,12 @@ export function draw(
   if (errorMsgContainer) {
     errorMsgContainer.innerHTML = "";
   }
-  const mainStatVar = props.statVarSpec.statVar;
+  // If multiple StatVars are provided, use the first
+  // TODO: Support multiple stat-vars
+  const mainStatVar = !_.isEmpty(props.dataSpec)
+    ? props.dataSpec[0].variable.statVar
+    : props.statVarSpec.statVar;
+  const unit = mainStatVar in chartData.units && chartData.units[mainStatVar];
```

Comment:
nit: can just do chartData.units[mainStatVar] because it will just be undefined if mainStatVar isn't in chartData.units

Reply:
Thanks for the pointer! Updated.